docs/manual/docs/examples/spo/adding-app-catalog.md

```diff
@@ -0,0 +1,18 @@
+# Hide SharePoint list from Site Contents
+
+Author: [David Ramalho](https://sharepoint-tricks.com/tenant-app-catalog-vs-site-collection-app-catalog/)
+
+
+When you just want to deploy certain SharePoint solution to a specific site, it's required to create an app catalog for that site, the below script will create it for the site. On the article link above you can check where you can use App catalog for the site instead of global app catalog.
+
+```powershell tab="PowerShell Core"
```

Comment:
When posting script samples we typically include both PowerShell and Bash variant. Would you like to try writing a bash version of it? If it's too far out of your comfort zone, totally understood and we can do it as well.

Reply:
It's not my conform zone but a skill that i can learn 😊 Both script have shell now 👍